app/helpers/application_helper.rb

```diff
@@ -1,2 +1,5 @@
 module ApplicationHelper
+  def is_ramsar_instance?
+    Rails.root.to_s.include?('API')
```

Comment:
don't we have a "stronger" check to see if the instance is a ramsar one??

Reply:
we do, it is in this commit https://github.com/unepwcmc/ORS-API/pull/16/commits/5f3d8640bb5fe58e10d8ff9e60b5b5b3219031a7. :sweat_smile: 
Please, forget my previous comment.